src/compiler/checker.ts

```diff
@@ -26058,6 +26060,16 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
                     // allowed template literal placeholder types, infer from a literal type corresponding to the constraint.
                     if (source.flags & TypeFlags.StringLiteral && target.flags & TypeFlags.TypeVariable) {
                         const inferenceContext = getInferenceInfoForType(target);
+                        // andarist here???

```

Comment:
Did you forget to remove/edit this?

Reply:
Yes 😅 i’ll remove this embarrassing leftover in a moment 😉